.msync.yml

```diff
@@ -2,4 +2,4 @@
 # Managed by modulesync - DO NOT EDIT
 # https://voxpupuli.org/docs/updating-files-managed-with-modulesync/
 
-modulesync_config_version: '9.5.0'
+modulesync_config_version: '9.7.0'
```

Comment:
Title and commit message says 9.6.0 but changes to 9.7.0
should title/commit message be changes to reflect this?